tests/e2e/page-utils.js

```diff
@@ -21,20 +21,37 @@ export async function elementExists( selector, root = page ) {
 }
 
 /**
- * Gets the text value of an element
+ * Gets the text value of all matched elements
  *
  * If the element is an `input` it will get the `value`, otherwise,
  * it will get the `textContent`.
  *
- * @param {string} selector The selector for the desired element
+ * @param {string} selector The selector for the desired elements
  * @param {Page | ElementHandle} [root=page] The root from which to search for the selector
  *
  * @return {Promise<string[]>} An array of text contained in those selected elements
  */
-export function getTextContent( selector, root = page ) {
+export function getTextContentAll( selector, root = page ) {
 	return root.$$eval( selector, ( $elements ) => {
 		return $elements.map(
 			( $element ) => $element.value || $element.textContent
 		);
 	} );
 }
```

Comment:
@sunyatasattva it's common to get the text content from a single element, so I think it's handier to have two versions of this function, one for multiple elements, and one for a single element. What do you think?

Reply:
I think a nice trick you can do to avoid having these two very similar functions is the following, change this:

```js
const priceInCart = await getTextContent(
  '.wc-block-cart-item__prices'
);
```

to this:

```js
const [ priceInCart ] = await getTextContent(
  '.wc-block-cart-item__prices'
);
```

Very readable, you avoid the ugly `( await getTextContent( '.wc-block-cart-item__prices' ) )[0]`, and we don't have to have two utility functions which do almost the same thing.

What do you think?

Reply:
@sunyatasattva It's a great trick! I updated my PR and added some examples to `getTextContent` including your destructuring trick above. Can you please take another look?